.env

```diff
@@ -0,0 +1 @@
+PHP_VERSION=8.0
```

Comment:
I may have missed something, but why is there a versionned .env file with an hardcoded PHP version while in Github actions, there is now a matrix of PHP versions?